### DISCUSSION TITLE
Is any release expected soon?

### DISCUSSION THREAD

**Discussion by galaxyfeeder** [aio-libs/aiohttp#6890] [Q&A] [answered]
Just wanted to know if there is any release coming soon or when are you looking to it.

More than half a year has passed from the last release, and there has been some bugfixes during this time.

Personally, I'm looking forward to having #6638 released. I'm pretty sure that there is other people out there waiting for some fixes to be released.

PS: Sorry, if this has already been answered in other places, I've checked Issues, discussions, gitter and discourse before posting.

**Comment by Dreamsorcerer** [comment, marked as answer]
It appears nobody backported that PR. If you have a moment to spare, you can follow the instructions to get it backported to the 3.9 release:
https://github.com/aio-libs/aiohttp/pull/6638#issuecomment-1061154506

Once @webknjaz is happy that Python 3.11 support is all good, then I think we'll do another 3.8 release with some fixes. Then I'd like to have a beta release for 3.9 following that, so we can get some feedback about the new `AppKey` feature before making a final release.

**Reply by galaxyfeeder** [reply]
Is there any discussion/issue on where the support for Python 3.11 is being carried on?

**Reply by galaxyfeeder** [reply]
About the backport, followed the instructions you have now a PR. https://github.com/aio-libs/aiohttp/pull/6891

**Reply by webknjaz** [reply]
It's https://github.com/aio-libs/aiohttp/issues/6600. Master is green but to unblock releasing, fixing the 3.8 and 3.9 is required. It's either me or someone else who should debug what's going on and send a PR fixing it.

**Reply by galaxyfeeder** [reply]
Just seen that this also wasn't backported to 3.8, I've just created the PR for it. #6974
